src/patternfly/components/FormControls/examples/form-controls-select-example.hbs

```diff
@@ -16,4 +16,14 @@
     <option value="Option 3">The third option</option>
     <option value="Option 4">The fourth option</option>
   </optgroup>
+{{/form-controls}}
+{{#> form-controls controlType="select" pf-c-form-control--modifier="pf-m-error" aria-invalid="true" aria-errormessage="incorrect text" id="select2" name="selectexample2" aria-label="select group example"}}
+  <optgroup label="Group 1">
+    <option value="Option 1">The first option</option>
```

Comment:
Need to add an option with a blank value for required select inputs.
`<option value="">make selection</option>`